Lesson15/Akan/script.js

```diff
@@ -0,0 +1,407 @@
+/*
+4. Extract Domain
+   - Define a function `getDomain(email)` that uses `slice` or `substring` to
+     extract everything after '@'.
+   - Log the domain to the console.
+   - Example: "user@example.com" -> "example.com"
+*/
+
+function getDomain(email) {
+  let domain = email.slice(email.indexOf('@') + 1);
+  console.log(domain);
+}
+
+/*
+5. Check Substring
+   - Define a function `containsWord(sentence, word)` that checks if the `sentence`
+     includes `word` (use the .includes() method).
+   - If true, log: "<word> found in sentence."
+   - Else, log: "<word> not found in sentence."
+*/
+
+function containsWorld(sentence, world) {
+  if (sentence.includes(world)) {
+    console.log(world + " found in sentence.");
+  }
+  else {
+    console.log(world + " not found in sentence.");
+  }
+}
+
```

Comment:
Function call is missing:
```suggestion
containsWord('This is my sentence', 'my');
```